Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Views/MainPage.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This MainPage seems to be a test or something similar. It is not used, could you delete it?.